website/docs/features/caching/index.md

```diff
@@ -178,6 +178,14 @@ request
 // Send the request
 ```
 
+You can also control the cache via JDBC:
```

Comment:
https://github.com/spicehq/team/blob/trunk/DOCUMENTATION.md#7-focus-on-the-technology-or-action